prow/plugins.yaml

```diff
@@ -84,6 +85,9 @@ plugins:
   - lifecycle
   - verify-owners
 
+  istio/operator:
+  - lgtm
```

Comment:
The standard tide block looks for both an approved and lgtm label (see L200 in config.yaml). So this repo will need OWNERS files and the approve plugin enabled.

If you're not wanting to do that then this repo will need its own section in tide. Make it stop looking for an approved label. If you want to make sure it gets reviewed by CODEOWNERS then you can set `reviewApprovedRequired: true` per 
 https://github.com/kubernetes/test-infra/blob/3f92da8679e28b626a91cf5f168ceee014d038f1/prow/config/tide.go#L181

Reply:
But note that essentially this is what Martin is planning to try out in the istio.io repo

Reply:
Ya, we have already transitioned to CODEOWNERS and off OWNERS.  I'll take a look at the link you posted and submit a followup.

Thanks!

Reply:
@fejta ya, @geeknoid asked me to trial this in the oeprator repo instead, as it might disrupt the automated backports that are implemented in mergify and widely used in the istio.io repo.